### PR TITLE
1251516: Disable import when registered

### PR DESCRIPTION
**Pull request by wottop**

### CODE REVIEW

**Comment by wottop** [comment]
buildbot: retest this please

File: src/subscription_manager/managercli.py [hunk 1]
```diff
@@ -1854,6 +1854,8 @@ def __init__(self):
                                help=_("certificate file to import (can be specified more than once)"))
 
     def _validate_options(self):
+        if self.is_registered():
+            system_exit(os.EX_USAGE, _("Error: You may not import certs into a system that is registered to a subscription management service."))
```
Comment (nguyenfilip) [inline src/subscription_manager/managercli.py]: I would use "certificates" instead of "certs". To be consistent with the rest of the subman code base

**Comment by nguyenfilip** [comment]
The PR looks good, tested on RHEL 7.2 in the following scenarios:

-  make sure its not possible to import certificate in registered mode
-  make sure its possible to imoprt certificate in un-registered mode

File: src/subscription_manager/managercli.py (hunk 1 above)
Comment (nguyenfilip) [inline src/subscription_manager/managercli.py]: @wottop Please just fix this, after that its good to merge.

File: test/test_managercli.py [hunk 2]
```diff
@@ -1171,9 +1171,19 @@ class TestImportCertCommand(TestCliCommand):
...
+        self.cc.is_registered = Mock(return_value=False)
         self.cc.main(["--certificate", "one", "--certificate", "two"])
         self.cc._validate_options()
 
+    def test_registered(self):
+        self.cc.is_registered = Mock(return_value=True)
+        self.cc.main(["--certificate", "one", "--certificate", "two"])
+        try:
+            self.cc._validate_options()
+            self.fail("No exception raised")
```
Comment (awood) [inline test/test_managercli.py]: you can make this clearer and shorter now with [assertRaises](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises)

```python
with self.assertRaises(SystemExit) as e:
    self.cc._validate_options()
self.assertEqual(os.EX_USAGE, e.exception.code)
```

**Pull request merged**